### PR TITLE
feat(TextField): allow addition of FEEL icon

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/bpmn-io/bpmn-js-properties-panel/issues/599

This PR adds the ability to refine the Input Field component to offer `Feel` support. We add an Icon when FEEL is either required or optional:

Optional:
Screenshot from 2022-03-10 10-44-35

Required:
Screenshot from 2022-03-10 10-43-47

Check out https://github.com/bpmn-io/bpmn-js-properties-panel/pull/602 for usage in the bpmn-js properties panel.

```
git clone git@github.com:bpmn-io/bpmn-js-properties-panel.git
cd bpmn-js-properties-panel
git checkout 599-feel-icons
npm ci
npm run start:cloud
```

### CODE REVIEW

File: src/components/entries/TextField.js [hunk 1]
```diff
@@ -11,6 +11,32 @@ import classnames from 'classnames';
 import {
   usePrevious
 } from '../../hooks';
+import { FeelOptionalIcon, FeelRequiredIcon } from '../icons';
+
+
+function FeelIcon(props) {
```
[the remark this answers is not in the document]
Comment (marstamm) [reply src/components/entries/TextField.js]: Done with [aea955f](https://github.com/bpmn-io/properties-panel/pull/138/commits/aea955f905248852d2dea3d6aeb859bbcc1ba708)

**Review by pinussilvestrus — APPROVED** [review]
Good job! 👏

**Comment by philippfromme** [comment]
Interesting. The most generic component, a text input, now knows about the concept of FEEL. Looking forward to refactoring this. 🤡

**Comment by marstamm** [comment]
This is interim anyway. When we move to active support (syntax Highlighting, suggestions, ...) we will need a FEEL component anyway :slightly_smiling_face:

**Pull request merged**